test/sql/weighted_search.sql

```diff
@@ -121,6 +120,30 @@ SELECT count(*)
   w3=> 0.52, col3=>'v_real'::text, vec3=>:'v444'::vector
 );
 
+-- Check if the type 'sparsevec' exists and store the result in a variable
+SELECT EXISTS (
```

Comment:
could we follow the current pattern and not do this in the test itself?

Maybe, see how we filter out pgvector or lantern extras tests when relevant extensions are not installed and do something similar here